roles/balloonprinter/templates/balloonprinter_run.sh.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Purpose: run coachview
+#
+
+export ROOTDIR=$( dirname "${BASH_SOURCE}[0]" )
+cd $ROOTDIR
+
+# additions for run
+DISPLAY=:0
+export DISPLAY
+umask 0077
+ulimit -c unlimited
+xset -dpms s off s noblank s 0 0 s noexpose
+gsettings set org.gnome.settings-daemon.plugins.power active false
+gsettings set org.gnome.desktop.screensaver idle-activation-enabled false
+gsettings set org.gnome.desktop.session idle-delay 0
```

Comment:
This can be set with ansible and is not specific to the balloonprinter?

Reply:
Would that be using community.general.dconf ?   We don't want to disable idle stuff on machines in general.

Reply:
Yes I think so,

And i think we want to disable idle stuff on all machines running ICPC tools? So mostly not on the contestantmachines? But that can be discussed in another PR.